connector/connect/client/jvm/src/main/scala/org/apache/spark/sql/streaming/StreamingQuery.scala

```diff
@@ -297,4 +298,20 @@ object RemoteStreamingQuery {
       name = if (result.getName.isEmpty) null else result.getName,
       sparkSession = sparkSession)
   }
+
+  def fromStreamingQueryInstanceResponse(
+      sparkSession: SparkSession,
+      q: StreamingQueryInstance): RemoteStreamingQuery = {
+
+    val name = if (q.hasName) {
+      q.getName
+    } else {
+      null
```

Comment:
Why null but not `Option`?

Reply:
Oh because `RemoteStreamingQuery` was not designed to take Option?

Reply:
Yes, also the original `StreamingQuery`. This is to align the behavior